app/versions/main-ui/routes.js

```diff
@@ -2,6 +2,7 @@ const express = require('express')
 const router = express.Router()
 const v1Routes = require('./v1/routes')
 const v2Routes = require('./v2/routes')
+const v3Routes = require('./v4/routes')
```

Comment:
This file looks like there's been a mis-merge. V3 and V4 have been mixed together.

Reply:
Nope. Just me missing places to update v3 to v4 :)